scripts/in_container/run_install_and_test_provider_packages.sh

```diff
@@ -38,6 +38,16 @@ if (($# < 1)); then
     exit 2
 fi
 
+if [[ ${AIRFLOW_VERSION} == *1.10* || ${INSTALL_AIRFLOW_VERSION} == *1.10* ]]; then
+    export RUN_AIRFLOW_1_10="true"
+    export CONSTRAINTS_BRANCH="constraints-master"
+else
+    export RUN_AIRFLOW_1_10="false"
+    export CONSTRAINTS_BRANCH="constraints-1-10"
```

Comment:
```suggestion
    export RUN_AIRFLOW_1_10="false"
    export CONSTRAINTS_BRANCH="constraints-master"
else
    export RUN_AIRFLOW_1_10="true"
    export CONSTRAINTS_BRANCH="constraints-1-10"
```
Or do I miss something?

Reply:
Obviously :D